server/src/test/java/org/opentosca/toscana/plugins/cloudfoundry/CloudFoundryApplicationTest.java

```diff
@@ -0,0 +1,106 @@
+package org.opentosca.toscana.plugins.cloudfoundry;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import org.opentosca.toscana.core.BaseUnitTest;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class CloudFoundryApplicationTest extends BaseUnitTest {
```

Comment:
In my opinion, this test class should be deleted. You are testing a data class -- a pojo -- a stupid POJO with no behavior. We should waste our time assuring the behavior of well tested standard features of java :)